onnxruntime/python/tools/transformers/models/whisper/README.md

```diff
@@ -75,6 +75,15 @@ $ python3 -m models.whisper.convert_to_onnx -m openai/whisper-tiny --output whis
 $ python3 -m onnxruntime.transformers.models.whisper.convert_to_onnx -m openai/whisper-tiny --output whispertiny --use_external_data_format --precision int8 --quantize_embedding_layer
```

Comment:
Could you replace example of whisper-tiny by whisper-large-v3 in README?  The later one is most downloaded model: https://huggingface.co/models?sort=downloads&search=whisper

Reply:
Fixed